README.md

```diff
@@ -83,3 +83,24 @@ STRIPE_ENDPOINT_SECRET_ACCOUNT="whsec_some-stripe-signing-secret"
 ## Stripe endpoint secrect for receiving webhooks from Stripe for endpoint-type "Connect" (required)
 (Webhook needs to be configured with stripe and given secret needs to be used)
 STRIPE_ENDPOINT_SECRET_CONNECT="whsec_some-stripe-signing-secret"
+
+# Development Setup
+
+To set up your development environment, run the following commands:
+
+```bash
+./deploy_local.sh
+```
```

Comment:
Oh, must be since I rebased. Will fix the dependancies to work again.

Reply:
Thanks you, @ChrisWeissmann ! 

In my case, I've installed python 3.12 on my Mac but it still shows me the `python3.10: command not found`, so it makes me feel that it not requires at least python 3.10 but exactly 3.10. I think [here](https://github.com/citrineos/citrineos-payment/blob/5f3b413e974cc1f5c42cfca1e47732d5522f54f3/deploy_local.sh#L2) cause this issue? 🤔

Reply:
@lydiazcheng can you try with `pyenv`? (https://github.com/pyenv/pyenv) if you are having python version issues, I have found `pyenv` to be very helpful. There may be better alternatives out there, but this is similar to Node's `nvm` and Ruby's `rvm` in that it helps you quickly switch between multiple versions of python installed on your machine. That error makes me think that it is still running the code with the wrong version and not finding 3.10 correctly...

@ChrisWeissmann I think we may need to add a `constraints.txt` file for `pip` to pick up along with `requirements.txt` to limit the version correctly since we are using pip?
```
# constraints.txt
python_version >= 3.7
```

Reply:
and you are right, that first line in the `.sh` file:
```
python3.10 -m venv venv
```
probably just needs to be:
```
python -m venv venv
```
and the `python` will resolve to correct version depending on the `PATH` env var I believe